bin/prompt_proto_upload_raws.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This file is part of prompt_prototype.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# This script uploads the raw files from the ap_verify_ci_cosmos_pdr2 repo
+# to Google Storage. It renames the files to match prompt_prototype conventions.
```

Comment:
Should note that the user has to have `setup ap_verify_ci_cosmos_pdr2` before this will work. I see there's an error message if it's not configured, but a note here would be good too.